bindings/pyroot/src/Utility.cxx

```diff
@@ -254,6 +254,27 @@ Bool_t PyROOT::Utility::AddToClass( PyObject* pyclass, const char* label, PyCall
    return kTRUE;
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// Add the given function with name 'func' to the class as read-only property 'property'.
+
+Bool_t PyROOT::Utility::AddProperty(PyObject *pyclass, const char *func, const char *property)
+{
+   PyObject *builtin = PyImport_ImportModule("__builtin__");
+   PyObject *property_class = PyObject_GetAttrString(builtin, "property");
+   PyObject *func_obj = PyObject_GetAttrString(pyclass, func);
```

Comment:
Short introduction to bound vs unbound methods in python:

```python
class A:
    def foo(self):
        return

A().foo # bound method, note the (): <bound method A.foo of <__main__.A instance at 0x7f491575ddd0>>
A.foo # unbound method: <unbound method A.foo>
```

Now, putting a print on the `func_obj` with `PyObject_Print(func_obj, stdout, 0);` returns the following:

**py2:** <unbound method TVectorT<float>._get__array_interface__>
**py3:** <bound method _get__array_interface__ of <NULL>>

So in short: The added method by the `Pythonize.cxx` does add an unbound `TVectorT.foo` method to the class for py2 while py3 adds a bound `TVectorT().foo` method pointing to a not existent instance of the class.

That's weird! I'll look further into this whether I've introduced this somewhere or whether this is a conceptual problem.